packages/forms-web-app/src/routes/file-based-router/comment-planning-appeal/add-comments/controller.js

```diff
@@ -0,0 +1,41 @@
+/** @type {import('express').RequestHandler} */
+const addCommentsGet = (req, res) => {
+	// if (!req.session.interestedParty.appealNumber) {
+	// To be discussed with design / team
+	// }
+
+	res.render(`comment-planning-appeal/add-comments/index`);
+};
+
+/** @type {import('express').RequestHandler} */
+const addCommentsPost = async (req, res) => {
+	// const { 'first-name': firstName, 'last-name': lastName } = req.body;
+
+	// if (!firstName) {
```

Comment:
This commented out stuff doesn't seem to reflect the form in the template for this page. Is the template changing or is this commented code not the right stuff?

Reply:
This is all placeholder at the moment as for now just needed to get the add-comments page to land on - the content and logic is being handled in a separate ticket but I'll tidy this one up too

Reply:
DW about clean up, I was just confuddled